include/roboteam_world/predictor.h

```diff
@@ -1,46 +0,0 @@
-#pragma once
```

Comment:
This file is as far as I know the last file that uses boost (maybe some utils class still uses it?)

Reply:
So we can probably remove it for good.